opentreemap/treemap/ecocache.py

```diff
@@ -50,12 +50,12 @@ def _get_or_compute(prefix, filter, compute_value):
 def _get_key(prefix, filter):
     if filter and (filter.filterstr or filter.displaystr):
```

Comment:
Here's a more concrete comment.

Example of why `eco_rev` is insufficient when a filter is active: You filter for only Oak trees. We compute and cache eco for your 25 oak trees. Now you update the species field of a tree from `null` to Oak. That doesn't change the `eco_rev`, so we incorrectly display the cached value instead of recomputing eco for your now 26 oak trees. Your update does change the `universal_rev` however, so using that in the cache key solves the problem.